docs/logic-ergo.md

```diff
@@ -12,7 +12,7 @@ Ergo aims to:
 - ensure safe execution: the language should prevent run-time errors and non-terminating logic
 - be blockchain neutral: the same contract logic can be executed either on and off chain on a variety of distributed ledger technologies
 - be formally specified: the meaning of contracts should be well defined so it can be verified, and preserved during execution
-- be consistent with the [Accord Project Template Specification](accordproject-specification)
+- be consistent with the [Accord Project Template Specification](ref-ergo-spec)
```

Comment:
Can you please change this to link to:
https://docs.accordproject.org/docs/accordproject-template.html

And modify the text to be

- be consistent with the Accord Project templates

Reply:
Ok sure :)